generated-platform-project/quarkus-qpid-jms/integration-tests/quarkus-qpid-jms-integration-tests/pom.xml

```
@@ -25,8 +25,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.quarkus</groupId>
+      <groupId>io.quarkiverse.artemis</groupId>
       <artifactId>quarkus-test-artemis</artifactId>
+      <version>${quarkus-amazon-services.version}</version>
```

```suggestion
      <version>${quarkus-artemis.version}</version>
```
and the property needs to be created.

Do you really need that though or was it just for convenience? Because that bugs me a bit to tie the platform to the Artemis repo.

---

I didnt do that, the build did. Thats the reason I raised the draft PR, I was commenting on it while you were hehe.

---

I used the test module since it makes sense not to duplicate it. It was in core, but was moved out to quarkiverse. I could duplicate it but it seems pointless.

---

@gsmet it's generated, we are going to have to accept it for now.